src/state/farms/elasticv2/updater.tsx

```diff
@@ -107,37 +108,61 @@ export default function ElasticFarmV2Updater({ interval = true }: { interval?: b
   const { networkInfo, isEVM, chainId, account } = useActiveWeb3React()
   const elasticFarm = useAppSelector(state => state.elasticFarmV2[chainId] || defaultChainData)
   const { elasticClient } = useKyberSwapConfig()
+  const isEnableKNProtocol = true
```

Comment:
is this temporary? @viet-nv

Reply:
LOL. That's for testing only. You guys missed it when review the code. LMaO

Reply:
Please help me to clear it

Reply:
Thanks a lot

Reply:
<img width="179" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/23650671/99104ffa-b93d-4649-b529-b30f09e66aad">

Reply:
![image](https://github.com/KyberNetwork/kyberswap-interface/assets/23650671/e890e96b-d7cb-4f2c-9516-00ad5e1ccb45)